frontend/src/metabase/dashboard/actions/cards-typed.ts

```diff
@@ -168,7 +208,10 @@ export const replaceCard =
     const dashboardId = getDashboardId(getState());
 
     let dashcard = getDashCardById(getState(), dashcardId);
-    if (isVirtualDashCard(dashcard)) {
+    if (
+      isVirtualDashCard(dashcard) &&
```

Comment:
Should we update `isVirtualDashCard` to consider placeholder cards as virtual ones?